pkg/java/v1/scaffolds/internal/templates/applicationproperties.go

```diff
@@ -0,0 +1,50 @@
+package templates
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"sigs.k8s.io/kubebuilder/v3/pkg/model/file"
+)
+
+var _ file.Template = &ApplicationPropertiesFile{}
+
+type ApplicationPropertiesFile struct {
+	file.TemplateMixin
+	OrgName         string
+	ProjectName     string
+}
+
+const (
+	FilePathSeparator = string(filepath.Separator)
+	javaPaths    = "src" + FilePathSeparator + "main" + FilePathSeparator + "resources"
```

Comment:
there are a bunch of extra spaces between `javaPaths` and the `=`.

Reply:
Modified according to suggestion